app/elements/services/profile-service.html

```diff
@@ -28,16 +28,19 @@
 					reflectToAttribute: true
 				}
 			},
+			ready: function(){
+				this.app = document.querySelector('#app');
```

Comment:
yes, with it we start the progress bar and currently polymer does not allow us to implement custom elements..